pom.xml

```diff
@@ -7,7 +7,7 @@
     </parent>
 
   <artifactId>gitlab-plugin</artifactId>
-  <version>1.1.29-SNAPSHOT</version>
+  <version>1.1.29.2-SNAPSHOT</version>
```

Comment:
No need for this change. When we do a release, the plugin version will increment automatically. So please revert this :)